tests/test_core.py

```diff
@@ -2362,6 +2362,9 @@ def test_stdlibs(self):
     self.set_setting('EXIT_RUNTIME')
     self.do_core_test('test_stdlibs.c')
 
+  def test_stdint_limits(self):
+    self.do_core_test('test_stdint_limits.c')
```

Comment:
This could be in other, I think, which is faster. Opt modes don't affect it?

Reply:
Done